packages/modal/modal-header.js

```diff
@@ -1,92 +1,111 @@
-import { CanCloseMixin } from './util.js'
-import WarpElement from '@warp-ds/elements-core'
-import { html, css, nothing } from 'lit'
-import { createRef, ref } from 'lit/directives/ref.js'
-import { modalV2 as cc } from './component-classes.js'
-import { Move } from '@itsy/animate/move'
+import { html, css, nothing } from 'lit';
 
-const NO_CLOSE_BUTTON = 'no-close'
+import { Move } from '@itsy/animate/move';
+import WarpElement from '@warp-ds/elements-core';
+import { createRef, ref } from 'lit/directives/ref.js';
+
+import { CanCloseMixin } from './util.js';
+
+const NO_CLOSE_BUTTON = 'no-close';
 
 export class ModalHeader extends CanCloseMixin(WarpElement) {
-  titleEl = createRef()
+  titleEl = createRef();
 
   static properties = {
     title: { type: String },
     back: { type: Boolean },
     [NO_CLOSE_BUTTON]: { type: Boolean },
     _hasTopContent: { state: true },
-  }
+  };
   constructor() {
-    super()
-    this._hasTopContent = false
+    super();
+    this._hasTopContent = false;
   }
   render() {
     return html`
-      <div class="${cc.titleSlot}">
+      <link href="https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css" rel="stylesheet" />
+      <div class="relative">
         <slot name="top" @slotchange=${this.handleTopSlotChange}></slot>
-        <div class="${!this._hasTopContent && cc.titleBarSlot}">
+        <div
+          class="${this._hasTopContent ? '' : 'pt-16 sm:pt-24 px-16 sm:px-32 grid gap-12 grid-cols-[auto_1fr_auto] items-start shrink-0!'}">
```

Comment:
Why a fake-ternary here? Unless this is a stylistic standard established in the Warp team, this is harder (to me) to parse than just using logical-and.

Lit is smart enough IIRC to handle falsey cases and won't put `null` or `false` as a class.

Reply:
We added this namely because we saw `class="false"` in the inspected DOM tree locally. So it seemed Lit didn't understand that `"false"` is not a class we want to assign when the left side of `&&` is `false` 😅

Reply:
You're right! I'm remembering wrong! 🙈